src/components/Fnr.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {FnrInput} from "nav-frontend-skjema";
+
+interface FnrProps {
+  label: string
+  nr?: string
```

Comment:
Bare for konsistens ville jeg foretrekke 'fnr' her

Reply:
Jeg gjør ingenting med denne da du pr tid legger denne inn i felles repo - som jeg skal bruke senere.

Reply:
Jeg gjorde denne alikevel :-)